models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -220,7 +220,8 @@ valued_trades as (
            limit_buy_amount,
            valid_to,
            flags,
-           case when (flags % 2) = 0 then 'SELL' else 'BUY' end as order_type
+           case when (flags % 2) = 0 then 'SELL' else 'BUY' end as order_type,
+           case when ((flags / 2) % 2) = 0 then 'Not partially fillable' else 'Partially fillable' end as partial_fill
```

Comment:
I am not in favour of using a string here. Would you mind making this Boolean? the column name should suffice as in indication of what it means.